homeassistant/components/zwave.py

```diff
@@ -371,10 +374,22 @@ def scene_activated(node, scene_id):
             ATTR_SCENE_ID: scene_id
         })
 
+    def node_event_activated(node, value):
+        """Called when a nodeevent is activated on any node in the network."""
+        name = _node_name(node)
+        object_id = "{}_{}".format(slugify(name), node.node_id)
```

Comment:
The existing _object_id(value) function is not supported by the scene and node event callbacks as the value parameter doesn't exist.  See the difference between _node_name and _value_name. 

I suppose a new function for _node_object_id(node) could be created if you'd rather it be done that way.